iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/Program.cs

```diff
@@ -47,9 +47,10 @@ public static async Task Main(string[] args)
                 });
             s_logger = loggerFactory.CreateLogger<Program>();
 
+            // Instantiating this seems to do all we need for outputting SDK events to our console log.
+            // The SDK logs are written at Trace level. To view SDK logs change the MinLogLevel to Trace.
             const string SdkEventProviderPrefix = "Microsoft-Azure-";
-            // Instantiating this seems to do all we need for outputting SDK events to our console log
-            _ = new ConsoleEventListener(SdkEventProviderPrefix, s_logger);
+            _ = new ConsoleEventListener(SdkEventProviderPrefix, logger);
```

Comment:
It seems that we didn't declare or use `logger` in the TemperatureController sample here, should we align this to `s_logger` instead?

Reply:
nice catch, copy-paste mistake!